builtin/logical/database/path_config_connection.go

```diff
@@ -0,0 +1,274 @@
+package database
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/fatih/structs"
+	"github.com/hashicorp/vault/builtin/logical/database/dbplugin"
+	"github.com/hashicorp/vault/helper/strutil"
+	"github.com/hashicorp/vault/logical"
+	"github.com/hashicorp/vault/logical/framework"
+)
+
+var (
+	respErrEmptyPluginName = logical.ErrorResponse("Empty plugin name")
```

Comment:
Errors across the API follow Go's lowercase standard -- we break this for CLI errors coming from the CLI itself as they tend to be more wordy.